### PR TITLE
Task

### PR DESCRIPTION
**Pull request by mmonge05**
# Adding changes

Is it possible to update the main with these fixes? 

Let us know your feedback.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -32,7 +32,7 @@
               Dropdown
             </a>
             <ul class="dropdown-menu" aria-labelledby="navbarDropdown">
-              <li><a class="dropdown-item" href="#">Action</a></li>
+              <li><a class="dropdown-item" href="#">Git Actions (CI)</a></li>
```
Comment (mmonge05) [inline index.html:35]: Hi Joan!

Hope you are doing well!! 

Do you agree to add Git Actions to the dropdown? @JoanEsquivel 

Let my know, please and if I apply the change!! 🥇

File: docs/index.html [hunk 2]
```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Document</title>
```
Comment (JoanEsquivel) [inline docs/index.html]: - [x] Please change the title according to your project.

File: index.html (hunk 1 above)
Comment (JoanEsquivel) [reply index.html:35]: Hello Moni, 

For next iterations, please assign the reviewer(in this case I did it). 

Hope you can check what I have suggested. 

Joan.

**Review by JoanEsquivel — CHANGES_REQUESTED** [review]

**Comment by JoanEsquivel** [comment]
Hello Moni, also seems that the branch has conflicts to resolve. Check it out.

File: index.html (hunk 1 above)
Comment (mmonge05) [reply index.html:35]: Hello! 

Thanks for this feedback I didn't know about "Reviewers" (A Fer se le fue!!) 🗡️

File: docs/index.html (hunk 2 above)
Comment (mmonge05) [reply docs/index.html]: Hi @JoanEsquivel 

The fix is ready to review! 

Thanks!

**Review by JoanEsquivel — APPROVED** [review]

**Comment by JoanEsquivel** [comment]
Hello Moni, 

Seems that "This branch has conflicts that must be resolved". We can not close this PR until this is fixed, please resolve it.

Thanks.

**Comment by mmonge05** [comment]
Hi @JoanEsquivel ,

The thing is this branch must overwrite the main branch. 
Can we do the merge/rebase on Friday?
In order to analyze the best scenario and possible conflicts, I need to merge all the changes to main.

I created 2 tags of the main and task branches and have the possibility to recover the info in the specific stage.

Thanks!

**Comment by JoanEsquivel** [comment]
Sure, we can review it. 

However, you can edit the conflicts you are having just clicking on the "resolve conflicts" button, and decide what you want to keep, delete, or save both. As soon as you just edit those changes, click on "Mark as resolved", and  then we can proceed with the "merge/rebase" strategy.

**Pull request merged**